failed test `UrlFormEncodingTests.UrlFormDecoderTests testBrackets`

sorry, don't have much time but just hit a failed test case, environment like #135 

```
Test Case '-[UrlFormEncodingTests.UrlFormDecoderTests testBrackets]' started.
swift-web/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift:77: error: -[UrlFormEncodingTests.UrlFormDecoderTests testBrackets] : failed - Diff: …

@@ −1,17 +1,17 @@
−▿ Foo #1 in UrlFormEncodingTests.UrlFormDecoderTests.testBrackets() throws -> ()
+▿ UrlFormEncodingTests.(unknown context at 0x1119d2b80).Foo
   - helloWorld: "a greeting for you"
   - port: 8080
   - bar: nil
   ▿ bars: 2 elements
−    ▿ Bar #1 in UrlFormEncodingTests.UrlFormDecoderTests.testBrackets() throws -> ()
+    ▿ UrlFormEncodingTests.(unknown context at 0x1119d2b44).Bar
       - baz: 1
−    ▿ Bar #1 in UrlFormEncodingTests.UrlFormDecoderTests.testBrackets() throws -> ()
+    ▿ UrlFormEncodingTests.(unknown context at 0x1119d2b44).Bar
       - baz: 2
   ▿ barses: 2 elements
     ▿ 1 element
−      ▿ Bar #1 in UrlFormEncodingTests.UrlFormDecoderTests.testBrackets() throws -> ()
+      ▿ UrlFormEncodingTests.(unknown context at 0x1119d2b44).Bar
         - baz: 3
     ▿ 1 element
−      ▿ Bar #1 in UrlFormEncodingTests.UrlFormDecoderTests.testBrackets() throws -> ()
+      ▿ UrlFormEncodingTests.(unknown context at 0x1119d2b44).Bar
         - baz: 4
```

thanks @stephencelis, oh wow, that Swift 4.2 regression is bad...

`UrlFormEncodingTests.UrlFormDecoderTests testBrackets` works well on `custom-snapshot-format` but when I run all the tests I now hit

```
Test Case '-[UrlFormEncodingTests.UrlFormDecoderTests testDateDecodingWithIso8601]' started.
/private/tmp/swift-web/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift:151: error: -[UrlFormEncodingTests.UrlFormDecoderTests testDateDecodingWithIso8601] : failed - Diff: …

@@ −1,4 +1,4 @@
 ▿ MyDate
−  ▿ date: 2017-12-12 01:36:00 +0000
+  ▿ date: 2017-12-12 1:36:00 am +0000
     - timeIntervalSinceReferenceDate: 534735360.0
```

which looks similar to #135 , probably needs more `en_US_POSIX`ing in other places ;)

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM swift:4.0
+FROM norionomura/swift:41
 
 WORKDIR /package
 

--- a/Package.resolved
+++ b/Package.resolved
@@ -77,8 +77,8 @@
         "package": "Prelude",
         "repositoryURL": "https://github.com/pointfreeco/swift-prelude.git",
         "state": {
-          "branch": "3b521bd",
-          "revision": "3b521bd5035bcda253ee9246cc025fc8b076a581",
+          "branch": "e159967",
+          "revision": "e1599670a962f678c8d21f3c400d33d582d6fe74",
           "version": null
         }
       },
@@ -86,8 +86,8 @@
         "package": "SnapshotTesting",
         "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing.git",
         "state": {
-          "branch": "08c0976",
-          "revision": "08c0976f968895cc2cfd07e843ca03c88dc11bac",
+          "branch": "d153af1",
+          "revision": "d153af19d04cacb53c310f09c1102c2450ed28b8",
           "version": null
         }
       },

--- a/Package.resolved
+++ b/Package.resolved
@@ -77,8 +77,8 @@
         "package": "Prelude",
         "repositoryURL": "https://github.com/pointfreeco/swift-prelude.git",
         "state": {
-          "branch": "e159967",
-          "revision": "e1599670a962f678c8d21f3c400d33d582d6fe74",
+          "branch": "8d597b0",
+          "revision": "8d597b051935dc7d63ec7cb6edefbeac07a73a94",
           "version": null
         }
       },
@@ -86,8 +86,8 @@
         "package": "SnapshotTesting",
         "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing.git",
         "state": {
-          "branch": "d153af1",
-          "revision": "d153af19d04cacb53c310f09c1102c2450ed28b8",
+          "branch": "2a0edb4",
+          "revision": "2a0edb4355793040ea1a80d580b42d1c651332c0",
           "version": null
         }
       },

--- a/Package.resolved
+++ b/Package.resolved
@@ -60,8 +60,8 @@
         "repositoryURL": "https://github.com/apple/swift-nio.git",
         "state": {
           "branch": null,
-          "revision": "91ef938c60061784ba8734a73d61b7810494f9b3",
-          "version": "1.5.0"
+          "revision": "77dc77b9b5cdddfb3c385c5ee6cb74153d284967",
+          "version": "1.7.2"
         }
       },
       {
@@ -77,8 +77,8 @@
         "package": "Prelude",
         "repositoryURL": "https://github.com/pointfreeco/swift-prelude.git",
         "state": {
-          "branch": "b811904",
-          "revision": "b8119040be900fbe6b32fd5c76be89bce7fd05b5",
+          "branch": "3b521bd",
+          "revision": "3b521bd5035bcda253ee9246cc025fc8b076a581",
           "version": null
         }
       },
@@ -86,8 +86,8 @@
         "package": "SnapshotTesting",
         "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing.git",
         "state": {
-          "branch": "0a86107",
-          "revision": "0a86107bae064e8429413dbedde1be1218fb683c",
+          "branch": "08c0976",
+          "revision": "08c0976f968895cc2cfd07e843ca03c88dc11bac",
           "version": null
         }
       },

--- a/Package.swift
+++ b/Package.swift
@@ -23,8 +23,8 @@ let package = Package(
     .library(name: "UrlFormEncoding", targets: ["UrlFormEncoding"]),
     ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("e159967")),
-    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("d153af1")),
+    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("8d597b0")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("2a0edb4")),
     .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
     .package(url: "https://github.com/bkase/DoctorPretty.git", .exact("0.4.1")),
     .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.22"))

--- a/Package.swift
+++ b/Package.swift
@@ -23,8 +23,8 @@ let package = Package(
     .library(name: "UrlFormEncoding", targets: ["UrlFormEncoding"]),
     ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("b811904")),
-    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("0a86107")),
+    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("3b521bd")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("08c0976")),
     .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
     .package(url: "https://github.com/bkase/DoctorPretty.git", .exact("0.4.1")),
     .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.22"))

--- a/Package.swift
+++ b/Package.swift
@@ -23,8 +23,8 @@ let package = Package(
     .library(name: "UrlFormEncoding", targets: ["UrlFormEncoding"]),
     ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("3b521bd")),
-    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("08c0976")),
+    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("e159967")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("d153af1")),
     .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
     .package(url: "https://github.com/bkase/DoctorPretty.git", .exact("0.4.1")),
     .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.22"))

--- a/Tests/LinuxMain.swift
+++ b/Tests/LinuxMain.swift
@@ -1,4 +1,4 @@
-// Generated using Sourcery 0.10.1 — https://github.com/krzysztofzablocki/Sourcery
+// Generated using Sourcery 0.11.0 — https://github.com/krzysztofzablocki/Sourcery
 // DO NOT EDIT
 
 import XCTest

--- a/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift
+++ b/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift
@@ -126,8 +126,9 @@ final class UrlFormDecoderTests: XCTestCase {
     }
 
     decoder.dateDecodingStrategy = .secondsSince1970
+    let interval = Int(Date(timeIntervalSinceReferenceDate: 0).timeIntervalSince1970)
 
-    assertSnapshot(matching: try decoder.decode(MyDate.self, from: Data("date=1513049223".utf8)))
+    assertSnapshot(matching: try decoder.decode(MyDate.self, from: Data("date=\(interval)".utf8)))
   }
 
   func testDateDecodingWithMillisecondsSince1970() throws {
@@ -145,14 +146,13 @@ final class UrlFormDecoderTests: XCTestCase {
     struct MyDate: Decodable {
       let date: Date
     }
-
     decoder.dateDecodingStrategy = .iso8601
 
     assertSnapshot(
-      matching: try decoder.decode(MyDate.self, from: Data("date=2017-12-11T20:36:00.000-05:00".utf8))
+      matching: try decoder.decode(MyDate.self, from: Data("date=2001-01-01T00:00:00.000-00:00".utf8))
     )
     assertSnapshot(
-      matching: try decoder.decode(MyDate.self, from: Data("date=2017-12-11T20:36:00-05:00".utf8))
+      matching: try decoder.decode(MyDate.self, from: Data("date=2001-01-01T00:00:00-00:00".utf8))
     )
   }
 

--- a/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift
+++ b/Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift
@@ -136,8 +136,9 @@ final class UrlFormDecoderTests: XCTestCase {
     }
 
     decoder.dateDecodingStrategy = .millisecondsSince1970
+    let interval = "\(Int(Date(timeIntervalSinceReferenceDate: 0).timeIntervalSince1970))000"
 
-    assertSnapshot(matching: try decoder.decode(MyDate.self, from: Data("date=1513049223123".utf8)))
+    assertSnapshot(matching: try decoder.decode(MyDate.self, from: Data("date=\(interval)".utf8)))
   }
 
   func testDateDecodingWithIso8601() throws {

